lib/stat_tracker.rb

```diff
@@ -1,38 +1,15 @@
-require 'csv'
-require_relative 'game'
-require_relative 'team'
-require_relative 'game_team'
-require_relative './game_processor'
-require_relative './league_processor'
-require_relative './season_processor'
-require_relative './team_processor'
-
-class StatTracker
+require_relative 'data_loader'
+
+class StatTracker < DataLoader
   include GameProcessor, LeagueProcessor, SeasonProcessor, TeamProcessor
   attr_reader :locations, :data
-  def initialize(game_path, team_path, game_teams_path)
```

Comment:
so much cleaner looking without all of this here, and still easily accessible to determine where the information is coming from